### ISSUE TITLE
Request: Alpine Linux support

### ISSUE THREAD

**Issue by lizelive** [NVIDIA/nvidia-docker#1606]
Alpine linux is very common in the container/docker ecosystem. It would make sense if `nvidia-docker` supported it as target.
Would need to be compiled against musl in order to do this.

**Comment by klueska** [comment]
I don't think it would take much work to get the nvidia-docker stack compiling with `musl`.
Unfortunately, this isn't sufficient to actually make it usable in any real scenarios.
The tools have an external dependce on `libnvidia-ml.so` which comes from the driver and is *not* compiled with musl.
Unless the NVIDIA driver starts supporting `musl` bases builds, there is not much we can do on the nvidia-docker front to support alpine-based distros.

**Comment by nvidiaLinuxUser** [comment]
+1

**Comment by henryjb99** [comment]
I would love to se3e Alpine Linux docker images be supported. Right now, it is a pain to not be able to run some of my programs in docker because of the fact that I cannot pass through the nvidia commands.
